pyannote/audio/core/model.py

```diff
@@ -468,9 +477,8 @@ def __by_name(
         if isinstance(modules, str):
             modules = [modules]
 
-        for name, module in ModelSummary(self, max_depth=-1).named_modules:
```

Comment:
Is `ModelSummary` still imported? 

On a related note, could you install pre-commit hooks like mentioned at the end of the README: https://github.com/pyannote/pyannote-audio#development

Among other things, it will automatically format your code and check for missing/unused imports every time you commit somehting.